### PR TITLE
triage updates; BRM triage

### PR DESCRIPTION
**Pull request by matebarabas**
# Overview/Summary

Triage updates and adding guidance for BRM triage

## This PR fixes/adds/changes/removes

see above

### Breaking Changes

n/a

## As part of this Pull Request I have

- [x] Read the Contribution Guide and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Verified-Modules/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Verified-Modules/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Verified-Modules/)
- [ ] Ensured PR tests are passing
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: utilities/repo-automation/brm/github/workflows/csv-label-sync.yml
Comment (jtracey93) [inline utilities/repo-automation/brm/github/workflows/csv-label-sync.yml:1]: Why this, the file paths wont work as not BRM specific

File: utilities/repo-automation/tf/github/policies/scheduledSearches.yml
Comment (jtracey93) [inline utilities/repo-automation/tf/github/policies/scheduledSearches.yml:1]: This should be PR'd into the TF repo template. As well as a label sync one and the label sync script

**Review by jtracey93 — CHANGES_REQUESTED** [review]
Can we also fix the index page for help and support not working 
image

**Comment by matebarabas** [comment]
Good catch on the navigation/menu items. I just fixed everything.

File: utilities/repo-automation/brm/github/workflows/csv-label-sync.yml
Comment (matebarabas) [reply utilities/repo-automation/brm/github/workflows/csv-label-sync.yml:1]: Correct, nothing was changed in this file (yet). The intent was to just move it here as a placeholder- to have an independent copy - for the time being. Then update it for BRM's requirements with you and decide if we want to keep a master copy here, or just the have the version living in BRM.

File: utilities/repo-automation/tf/github/policies/scheduledSearches.yml
Comment (matebarabas) [reply utilities/repo-automation/tf/github/policies/scheduledSearches.yml:1]: Similar idea as mentioned above. The intent is to have a master copy before we place a copy into the repo template (this file was however updated to reflect Terraform's requirements).

File: utilities/repo-automation/brm/github/workflows/csv-label-sync.yml
Comment (jtracey93) [reply utilities/repo-automation/brm/github/workflows/csv-label-sync.yml:1]: i vote to just have it BRM, we dont need 2 sources of truth 👍

File: utilities/repo-automation/tf/github/policies/scheduledSearches.yml
Comment (jtracey93) [reply utilities/repo-automation/tf/github/policies/scheduledSearches.yml:1]: same as other comment, only have it in the template repo

**Review by jtracey93 — CHANGES_REQUESTED** [review]
lets remove the files that dont need to be in here and then we can merge it

**Review by jtracey93 — APPROVED** [review]
LGTM

**Pull request merged**